src/utils/contacts.js

```diff
@@ -20,14 +20,14 @@
 
 import get from 'lodash.get';
 import { STATUS_BLOCKED } from 'constants/connectionsConstants';
+import Toast from 'components/Toast';
 import type { ApiUser, ContactSmartAddressData } from 'models/Contacts';
 import { addressesEqual } from './assets';
-import { isCaseInsensitiveMatch } from './common';
+import { isCaseInsensitiveMatch, resolveEnsName } from './common';
+import { isEnsName } from './validators';
 
 export const getUserName = (contact: ?Object) => {
-  if (!contact || !contact.username) {
-    return '';
-  }
+  if (!contact || !contact.username) return '';
```

Comment:
I'm starting to wonder, when you specify `username` are we actually talking about the *ENS* name?

Reply:
Nope, here username is username of the (now legacy) contact. I'm removing it as no longer necessary.